### PR TITLE
add new tags in CLI

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
add new tags in CLI

**How to test add new tags in CLI**:
1. install on stage: `bash /home/proj/stage/servers/resources/hasta.scilifelab.se/update-housekeeper-stage.sh feat/add-new-tag`
1. activate stage: `us`
1. run following command: `housekeeper add tag "coffee"`

**Expected outcome**:
`new tag added: coffee (NNNN)` where NNNN is the id of the new tag
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @henrikstranneheim 
- [x] tests executed by @henrikstranneheim 
- [x] "Merge and deploy" approved by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This is minor **version bump** because we added a feature (file needs to be specified with -f and is now optional)

### CODE REVIEW

**Comment by emiliaol** [comment]
Doesn't this functionality already exist? Or maybe I don't get what you want to do.

**Comment by henrikstranneheim** [comment]
**How to test add new tags in CLI**:
1. run following command: `housekeeper add tag "coffee"`

**Expected outcome**:
`new tag added: coffee (NNNN)` where NNNN is the id of the new tag **FAIL**

Screenshot 2020-02-10 at 15 04 25

**Comment by henrikstranneheim** [comment]
@patrikgrenfeldt The test is failing could you take a look?

**Comment by patrikgrenfeldt** [comment]
I hope I fixed it. you are welcome to tests again @henrikstranneheim (but I haven't tested myself :/ !)

File: housekeeper/cli/add.py [hunk 1]
```diff
@@ -58,23 +58,38 @@ def file_cmd(context, tags, archive, bundle_name, path):
...
+@click.option('-f', '--file-id' , 'file_id', type=int)
 @click.pass_context
-def tag(context: click.Context, file_id: int, tags: List[str]):
+def tag(context: click.Context, tags: List[str], file_id: int=None):
     """Add tags to an existing file."""
-    file_obj = context.obj['db'].file_(file_id)
-    if file_obj is None:
-        print(click.style('unable to find file', fg='red'))
-        context.abort()
+
```
Comment (henrikstranneheim) [inline housekeeper/cli/add.py]:
```suggestion
file_obj = None

```

File: housekeeper/cli/add.py [hunk 2]
```diff
@@ -58,23 +58,39 @@ def file_cmd(context, tags, archive, bundle_name, path):
...
+@click.option('-f', '--file-id' , 'file_id', type=int)
 @click.pass_context
-def tag(context: click.Context, file_id: int, tags: List[str]):
+def tag(context: click.Context, tags: List[str], file_id: int=None):
     """Add tags to an existing file."""
-    file_obj = context.obj['db'].file_(file_id)
-    if file_obj is None:
-        print(click.style('unable to find file', fg='red'))
-        context.abort()
+file_obj = None
```
Comment (henrikstranneheim) [inline housekeeper/cli/add.py]:
```suggestion
    file_obj = None
```

**Comment by henrikstranneheim** [comment]
@patrikgrenfeldt If fixed the code and now the tests pass. Please have a look

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**